packages/exporter-trace-otlp-grpc/src/OTLPExporterNodeBase.ts

```diff
@@ -46,14 +46,16 @@ export abstract class OTLPExporterNodeBase<
 
   constructor(config: OTLPExporterConfigNode = {}) {
     super(config);
-    if (config.headers) {
-      diag.warn('Headers cannot be set when using grpc');
-    }
     const headers = baggageUtils.parseKeyPairsIntoRecord(getEnv().OTEL_EXPORTER_OTLP_HEADERS);
     this.metadata = config.metadata || new Metadata();
     for (const [k, v] of Object.entries(headers)) {
       this.metadata.set(k, v)
     }
+    if (config.headers) {
```

Comment:
I think you can directly give `config.metadata` if you want to send specific values

Reply:
You are right. I missed that.